### ISSUE TITLE
[V3 Data Converter] Alias support

### ISSUE THREAD

**Issue by tekulvw** [Cog-Creators/Red-DiscordBot#1492]
### Type:

- Suggestion

### Brief description of the problem
The newly introduced data converter is currently lacking alias support.

**Comment by mikeshardmind** [comment]
This needs to be incorporated differently to validate the existence of commands and prompt on lack. putting it on my to-do list.

**Comment by mikeshardmind** [comment]
Would the preferred method here be to just fail on any command that isn't identical, or should I build a lookup dict with before and after of commands (v2 -> v3)?

**Comment by mikeshardmind** [comment]
This needs to be implemented after #1749 to ensure that the conversions happen based on the names of commands then.

**Comment by tekulvw** [comment]
Dataconverter no longer exists, this is no longer needed.
